<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionMemory"/>
    <addaction name="actionProgram_memory"/>
    <addaction name="actionCache"/>
    <addaction name="actionCache_statistics"/>
    <addaction name="actionRegisters"/>
   </widget>
   <widget class="QMenu" name="menuMachine">
    <property name="title">
     <string>Machine</string>
    </property>
    <widget class="QMenu" name="menuSimulation_speed">
     <property name="title">
      <string>Simulation speed</string>
     </property>
     <addaction name="action1_ips"/>
     <addaction name="action2_ips"/>
     <addaction name="action5_ips"/>
     <addaction name="action10_ips"/>
     <addaction name="actionOther_ips"/>
     <addaction name="separator"/>
     <addaction name="actionUnlimited"/>
    </widget>
    <addaction name="actionRun"/>
    <addaction name="actionPause"/>
    <addaction name="actionStep"/>
    <addaction name="menuSimulation_speed"/>
    <addaction name="separator"/>
    <addaction name="actionRestart"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMachine"/>
   <addaction name="menuWindows"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionReload"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionPause"/>
   <addaction name="actionStep"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/document-import.png</normaloff>:/icons/document-import.png</iconset>
   </property>
   <property name="text">
    <string>New simulation...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O, Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/pause.png</normaloff>:/icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action1_ips">
   <property name="text">
    <string>1 instruction per second</string>
   </property>
  </action>
  <action name="action2_ips">
   <property name="text">
    <string>2 instructions per second</string>
   </property>
  </action>
  <action name="action5_ips">
   <property name="text">
    <string>5 instructions per second</string>
   </property>
  </action>
  <action name="action10_ips">
   <property name="text">
    <string>10 instructions per second</string>
   </property>
  </action>
  <action name="actionUnlimited">
   <property name="text">
    <string>Unlimited</string>
   </property>
  </action>
  <action name="actionOther_ips">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Custom...</string>
   </property>
  </action>
  <action name="actionMachine_control">
   <property name="text">
    <string>Machine control</string>
   </property>
  </action>
  <action name="actionMemory">
   <property name="text">
    <string>Data Memory</string>
   </property>
  </action>
  <action name="actionProgram_memory">
   <property name="text">
    <string>Program memory</string>
   </property>
  </action>
  <action name="actionCache">
   <property name="text">
    <string>Cache content</string>
   </property>
  </action>
  <action name="actionCache_statistics">
   <property name="text">
    <string>Cache statistics</string>
   </property>
  </action>
  <action name="actionRegisters">
   <property name="text">
    <string>Registers</string>
   </property>
  </action>
  <action name="actionAssembler">
   <property name="text">
    <string>Assembler</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/reload.png</normaloff>:/icons/reload.png</iconset>
   </property>
   <property name="text">
    <string>Reload existing</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
